### PR TITLE
update C0DE-OF-CONDUCT.md

### PR DESCRIPTION
**Pull request by domitechnshimba**
update C0DE-OF-CONDUCT.md

Note that I have made a few changes to MD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <img src="images/GitHub-Universe-logo-2020.svg" width="300" alt="GitHub Universe Virtual 2020">
 
-[https://githubuniverse.com/](https://githubuniverse.com/)
+Visit the Officical Weboste: [https://githubuniverse.com](https://githubuniverse.com/)
```
Comment (Flaxxas) [inline README.md:3]:
```suggestion
Visit the official website: [https://githubuniverse.com](https://githubuniverse.com/)
```

**Comment by xetra11** [comment]
I feel better now being on Github

File: README.md (hunk 1 above)
Comment (Lubo-mir) [inline README.md:3]:
```suggestion
Visit the Officical Website: [githubuniverse.com](https://githubuniverse.com/)
```
Protocol and path prefix are not necessary to display 😉

**Review by Lubo-mir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HPaulson) [inline README.md:3]: Why not just use a hyperlink here?
```suggestion
Visit the [Official Website](https://githubuniverse.com/)
```

**Review by HPaulson — CHANGES_REQUESTED** [review]

**Comment by martinwoodward** [comment]
Thanks folks - probably just leave this one as is for now but appreciate you pointing out the potential fixes.

**Pull request closed without merging**